Prog/NC target tables tabulation.sas

```diff
@@ -0,0 +1,537 @@
+    1 = '0-30% AMI'
+    2 = '31-50%'
+    3 = '51-80%'
+    4 = '81-120%'
+    5 = '120-200%'
+    6 = 'More than 200%'
+	;
+
+  value tenure
+    1 = 'Renter units'
+    2 = 'Owner units'
+	;
+/*
+  value county2_char
+    1= "DC"
+	2= "Charles County"
+	3= "Frederick County "
+	4="Montgomery County"
+	5="Prince Georges "
+	6="Arlington"
+	7="Fairfax, Fairfax city and Falls Church"
+	8="Loudoun"
+	9="Prince William, Manassas and Manassas Park"
+    10="Alexandria"
+  	;
+*/
+  value rcost
+	  1= "$0 to $749"
+	  2= "$750 to $1,199"
+	  3= "$1,200 to $1,499"
+	  4= "$1,500 to $1,999"
+	  5= "$2,000 to $2,499"
+	  6= "More than $2,500"
+  ;
+
+  value ocost
+	  1= "$0 to $1,199"
+	  2= "$1,200 to $1,799"
+	  3= "$1,800 to $2,499"
+	  4= "$2,500 to $3,199"
+	  5= "$3,200 to $4,199"
+	  6= "More than $4,200"
+  ;
+
+  value acost
+	  1= "$0 to $799"
+	  2= "$800 to $1,299"
+	  3= "$1,300 to $1,799"
+	  4= "$1,800 to $2,499"
+	  5= "$2,500 to $3,499"
+	  6= "More than $3,500"
+  ;
+	
+  value inc_cat
+
+    1 = '20 percentile'
+    2 = '40 percentile'
+    3 = '60 percentile'
+	4 = '80 percentile'
+	5= '100 percentile'
+    6= 'vacant'
+	;
+  value structure
+	1= 'Single family attached and detached'
+	2= '2-9 units in strucutre'
+	3= '10+ units in strucutre'
+	4= 'Mobile or other'
+	5= 'NA'
+	;
+  	
+  value afford
+
+  1= 'natural affordable (rent < $750)'
+  0= 'not natural affordable'; 
+run;
+
+/*read in dataset created by NCHousing_needs_units_targets.sas*/
+data fiveyeartotal;
+	set NCHsg.fiveyeartotal ;
+run;
+
+ data fiveyeartotal_vacant; 
+   set NCHsg.fiveyeartotal_vacant;
+ run;
+
+ data fiveyeartotal_othervacant; 
+   set NCHsg.fiveyeartotal_othervacant ;
+ run;
+
+proc tabulate data=fiveyeartotal format=comma12. noseps missing;
+  class county2_char;
+  var hhwt_5;
+  table
+    all='Total' county2_char=' ',
+    sum='Sum of HHWTs' * ( hhwt_5='Original 5-year'  )
+  / box='Occupied housing units';
+  *format county2_char county2_char.;
+run;
+
+proc tabulate data=fiveyeartotal_vacant format=comma12. noseps missing;
+  class county2_char;
+  var hhwt_5;
+  table
+    all='Total' county2_char=' ',
+    sum='Sum of HHWTs' * ( hhwt_5='Original 5-year')
+  / box='Vacant (nonseasonal) housing units';
+  *format county2_char county2_char.;
+run;
+
+proc tabulate data=fiveyeartotal_othervacant format=comma12. noseps missing;
+  class county2_char;
+  var hhwt_5;
+  table
+    all='Total' county2_char=' ',
+    sum='Sum of HHWTs' * ( hhwt_5='Original 5-year' )
+  / box='Seasonal vacant housing units';
+  *format county2_char county2_char.;
+run;
+
+proc freq data=fiveyeartotal_othervacant;
+by county2_char;
+tables vacancy /nopercent norow nocol out=other_vacant;
+weight hhwt_5;
+*format county2_char county2_char.;
+run; 
+proc export data=fiveyeartotal_othervacant
+ 	outfile="&_dcdata_default_path\NCHsg\Prog\other_vacant_&date..csv"
+   dbms=csv
+   replace;
+   run;
+
+/*data set for all units that we can determine cost level*/ 
+data all;
```

Comment:
please create "all" in the other program and add a data set label and proc contents.